tests/expr2.f90

```diff
@@ -0,0 +1,9 @@
+program expr2
+implicit none
+
+integer :: x
+
+x = (2+3)*5
+print *, x
```

Comment:
Does `error stop` print a stacktrace?

Reply:
Yes, it seems to print on both linux and macos.

Reply:
Perfect!